spec/ralyxa/response_entities/directives/audio_player_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../../spec_helper'
+
+RSpec.describe Ralyxa::ResponseEntities::Directives::AudioPlayer do
+  describe 'self.' do
+    describe '#play' do
```

Comment:
This reads a little funny in the documentation output – prefer `.play` if possible!